tests/src/test/scala/org/http4s/EntityDecoderSpec.scala

```diff
@@ -301,7 +301,7 @@ class EntityDecoderSpec extends Http4sSpec with Http4sLegacyMatchersIO with Pend
 
     "invoke the function with  the right on a success" in {
       val happyDecoder: EntityDecoder[IO, String] =
-        EntityDecoder.decodeBy(MediaRange.`*/*`)(_ => DecodeResult.success(IO.pure("hooray")))
+        EntityDecoder.decodeBy(MediaRange.`**`)(_ => DecodeResult.success(IO.pure("hooray")))
```

Comment:
Is this change to make it compile inside the block comment?  You can put another `*/` at the end, which looks imbalanced, but works.

Or at least you could in Scala 2.9 or whatever when I first had this problem. :laughing: